MapboxNavigation/StyleManager.swift

```diff
@@ -96,6 +96,21 @@ open class StyleManager: NSObject {
         resetTimeOfDayTimer()
     }
     
+    func applyStyle(type styleType: StyleType) {
```

Comment:
Could this func be combined with the `applyStyle` below?

Reply:
Yes, I considered it. However, this func `applyStyle(type:)` is cleaner, since it requires the `styleType`. If you notice, the other func `applyStyle ` scope is getting a bit bigger and complex with more conditions.

Reply:
`UPDATES:` I also refactored `applyStyle` so that we have a function  `applyStyle(type:)` to contain `styleType` related logic.